Lab3.ipynb

```diff
@@ -0,0 +1,405 @@
+      "text/plain": [
+       "<PIL.JpegImagePlugin.JpegImageFile image mode=RGB size=202x49 at 0x7F40B7492850>"
+      ]
+     },
+     "execution_count": 120,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# 3.2:\n",
+    "print(\"Question to math\")\n",
+    "Image.open('Lab3.3.jpg')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 121,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Probability: 1.0133280183917768e-06\n",
+      "Sigma: -4.750747965777188\n"
+     ]
+    }
+   ],
+   "source": [
+    "# 3.3: \n",
+    "prob = np.abs(stats.poisson.cdf(6800, mu=7200))\n",
+    "sigma = stats.norm.ppf(prob)\n",
+    "print(\"Probability:\", prob)\n",
+    "print(\"Sigma:\", sigma)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Problem 3.3: The significance of the measurement is 4.75 sigma."
```

Comment:
Problem 3.3
- makes sense based on the math, but I would connect this back to the statistical question/problem a little further.